spicy/toolchain/src/compiler/parser/parser.yy

```diff
@@ -921,6 +922,13 @@ unit_switch_case
               | ARROW unit_field                 { $$ = builder->typeUnitItemSwitchCase($2, __loc__); }
               | ARROW unit_switch                { $$ = builder->typeUnitItemSwitchCase($2, __loc__); }
 
+opt_unit_block_else_items
+              : ELSE '{' unit_items '}'          { $$ = std::move($3); }
+              | /* empty */                      { $$ = {}; }
+
+unit_block    : IF '(' expr ')' '{' unit_items '}' opt_unit_block_else_items ';'
```

Comment:
WDYT about switching to `opt_unit_items` here to allow empty blocks, e.g., for commenting easier commenting out during development? This would mirror what is allowed for procedural `if`.